applications/nrf5340_audio/sample.yaml

```diff
@@ -10,10 +10,34 @@ tests:
     platform_allow: nrf5340_audio_dk_nrf5340_cpuapp
     platform_exclude: nrf5340_audio_dk_nrf5340_cpuapp_ns
     tags: ci_build
-    extra_args: CMAKE_BUILD_TYPE=RELEASE DEV_HEADSET=ON
+    extra_args: CONF_FILE="prj_release.conf" CONFIG_AUDIO_DEV=1
```

Comment:
Just for my understanding, what is the reason for changing the format of extra_args?

Reply:
Those old extra_args does no longer effect since we were refactored the build script.
Thus, I changed the format to make those cases working.